code/modules/cargo/exports/sheets.dm

```diff
@@ -35,7 +35,7 @@
 /datum/export/goliath_hide
 	cost = 2500
 	unit_name = "goliath hide"
-	export_types = list(/obj/item/asteroid/goliath_hide)
+	export_types = list(/obj/item/stack/sheet/animalhide/goliath_hide)
```

Comment:
Change `/datum/export/goliath_hide` to `/datum/export/stack/skin/goliath_hide`. This would make it calculate the export cost correctly.